pyproject.toml

```diff
@@ -11,7 +11,7 @@ requires-python = '>=3.7'
 dependencies = [
         'tqdm',
         'pdf2image',
-        'pdfplumber>0.7.1',
+        'pdfplumber==0.7.4',
```

Comment:
Any reason to pin to 0.7.4 instead of 0.7.6?

Reply:
0.7.6 upgrades a version of `pdfminer` under the hood where the `pdfminer` logging isn't done properly & it floods our logs w/ a bunch of extra msgs. really hard to debug things & it doesnt look like any useful functionality diff from 0.7.4 to 0.7.6